### PR TITLE
UIREQ-938 Create Jest\RTL test for RequestsFiltersConfig.js

### PR DESCRIPTION
**Pull request by VSnehalatha**
Create Jest/RTL test for RequestsFiltersConfig.js
Refs: [UIREQ-938](https://issues.folio.org/browse/UIREQ-938)

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

File: src/components/RequestsFilters/RequestsFiltersConfig.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { requestFilterTypes } from '../../constants';
+import filtersConfig from './RequestsFiltersConfig';
+
+describe('RequestsFiltersConfig', () => {
+  it('should have a filter for requestType', () => {
+    const requestTypeFilter = filtersConfig.find(f => f.name === 'requestType');
+    expect(requestTypeFilter).toBeTruthy();
+    expect(requestTypeFilter.cql).toEqual('requestType');
+    expect(requestTypeFilter.values).toEqual([]);
+    expect(requestTypeFilter.operator).toEqual('==');
```
Comment (artem-blazhko) [inline src/components/RequestsFilters/RequestsFiltersConfig.test.js:10]: Could you change those "expect" statements to only one "expect"?
```suggestion
    const expectedResult = {
    name: 'requestType',
    cql: 'requestType',
    values: [],
    operator: '==',
    };

    expect(requestTypeFilter).toEqual(expectedResult);
```

Could you do the same with other tests?

File: src/components/RequestsFilters/RequestsFiltersConfig.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  });
+
+  it('should have a filter for tags', () => {
+    const tagsFilter = filtersConfig.find(f => f.name === 'tags');
+    expect(tagsFilter).toBeTruthy();
+    expect(tagsFilter.cql).toEqual('tags.tagList');
+    expect(tagsFilter.values).toEqual([]);
+    expect(tagsFilter.operator).toEqual('==');
+    expect(tagsFilter.parse('tag1')).toEqual('tags.tagList==*"*tag1*"*');
+    expect(tagsFilter.parse(['tag1', 'tag2'])).toEqual('tags.tagList==(*"*tag1*"* or *"*tag2*"*)');
```
Comment (artem-blazhko) [inline src/components/RequestsFilters/RequestsFiltersConfig.test.js:37]: Could you create 2 separate tests to check "parse" method?

**Review by artem-blazhko — CHANGES_REQUESTED** [review]
Could you add appropriate changes to CHANGELOG.md file?

File: src/components/RequestsFilters/RequestsFiltersConfig.test.js (hunk 1 above)
Comment (VSnehalatha) [reply src/components/RequestsFilters/RequestsFiltersConfig.test.js]: All tests updated

File: src/components/RequestsFilters/RequestsFiltersConfig.test.js (hunk 2 above)
Comment (VSnehalatha) [reply src/components/RequestsFilters/RequestsFiltersConfig.test.js]: Created 2 separate tests, please check.

File: src/components/RequestsFilters/RequestsFiltersConfig.test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    const tagsFilter = filtersConfig.find(f => f.name === 'tags');
+    const expectedResult = {
+      name: 'tags',
+      cql: 'tags.tagList',
+      values: [],
+      operator: '==',
+      parse: expect.any(Function),
+    };
+    expect(tagsFilter).toEqual(expectedResult);
+    expect(tagsFilter.parse('tag1')).toEqual('tags.tagList==*"*tag1*"*');
```
Comment (artem-blazhko) [inline src/components/RequestsFilters/RequestsFiltersConfig.test.js:49]: Could you create two separate tests for each `expect`?

Comment (VSnehalatha) [reply src/components/RequestsFilters/RequestsFiltersConfig.test.js]: Updated the test

**Review by artem-blazhko — APPROVED** [review]

**Review by Dmitriy-Litvinenko — APPROVED** [review]
Hello. Thank you. Could you please update or rebase branch and merge it?

**Pull request merged**